eng/common/pipelines/templates/steps/bypass-local-dns.yml

```diff
@@ -6,6 +6,9 @@ steps:
       condition: |
         and(
           succeededOrFailed(),
-          eq(variables['OSVmImage'], 'ubuntu-18.04'),
+          or(
+            eq(variables['OSVmImage'], 'ubuntu-18.04'),
+            eq(variables['OSVmImage'], 'MMSUbuntu18.04')
```

Comment:
@mitchdenny do we have this set for all the pipelines now? OR will this still be an issue for some that don't have an OSVmImage set yet?

Reply:
I think if this was working before, there's no reason to think it will break now with the addition of an extra variable check. If the variable doesn't exist it will just evaluate to false.

Reply:
We still have some pipelines using ubuntu-18.04, specifically legs within the release side of the pipelines. So having it cover both scenarios is useful for now. And possibly going to be useful in the future when we start using Ubuntu 20.04 etc.

Reply:
I was fine with covering both. My question was more around did we update all the new pipelines with the new name, I know there were some where the OSVmImage name was empty and I'm not sure if you updated all those yet or not.

Reply:
Ah I see